### PR TITLE
Base Template for all module docs

### PR DESCRIPTION
**Pull request by yevgenybulochnik**
Consistent template for all module docs, I figure if someone wants to fill out a doc for a module they should create a branch for a specific doc and open a PR. What do you guys think

### CODE REVIEW

**Comment by jerrywardlow** [comment]
If we're going to create module docs on a one-by-one basis, I would recommend the branch naming scheme `doc/module/<module-name>` for consistency.

File: docs/modules/haproxy.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# HaProxy
```
Comment (jerrywardlow) [inline docs/modules/haproxy.md]: HAProxy

File: docs/modules/logstash.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# LogStash
```
Comment (jerrywardlow) [inline docs/modules/logstash.md]: Logstash

File: docs/modules/mongodb.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# MongDB
```
Comment (jerrywardlow) [inline docs/modules/mongodb.md]: MongoDB

File: docs/modules/wikijs.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# WikiJS
```
Comment (jerrywardlow) [inline docs/modules/wikijs.md]: Wiki.js

**Comment by yevgenybulochnik** [comment]
That makes sense.

File: docs/modules/elasticsearch.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# ElasticSearch
```
Comment (jerrywardlow) [inline docs/modules/elasticsearch.md]: Elasticsearch

**Review by jerrywardlow — CHANGES_REQUESTED** [review]
Request changes on headings to maintain formatting consistency with organization

**Review by jerrywardlow — APPROVED** [review]

**Pull request merged**